server/remote/gcal/get_schedule.go

```diff
@@ -0,0 +1,9 @@
+package gcal
+
+import (
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+func (c *client) GetSchedule(_ []*remote.ScheduleUserInfo, _, _ *remote.DateTime, _ int) ([]*remote.ScheduleInformation, error) {
```

Comment:
This is actually dead code for mscalendar as well. It's implemented but not called anywhere IIRC

Reply:
Yeah, I noticed that but wasn't sure if it was a feature in development or something, so I wanted to be safe about it.